sim/battle-actions.ts

```diff
@@ -1668,9 +1668,10 @@ export class BattleActions {
 		}
 
 		if (
-			basePower < 60 && source.getTypes(true).includes(move.type) && source.terastallized && move.priority <= 0 &&
+			basePower < 60 && source.getTypes(true).includes(move.type) && source.terastallized &&
+			this.dex.moves.get(move.id).priority <= 0 && !move.multihit &&
```

Comment:
I would also get the move's true multihit status from move data while you're at it; Tera Electric Parental Bond Nuzzle, for example, will be 60 BP both times.